perun/profile/imports.py

```diff
@@ -0,0 +1,200 @@
+"""Functions for importing Profile from different formats"""
+
+from __future__ import annotations
+
+# Standard Imports
+from typing import Any, Optional
+import json
+import os
+import subprocess
+
+# Third-Party Imports
+
+# Perun Imports
+from perun.collect.kperf import parser
+from perun.profile import helpers as profile_helpers
+from perun.logic import commands, index, pcs
+from perun.utils import log, streams
+from perun.utils.common import script_kit
+from perun.utils.external import commands as external_commands, environment
+from perun.utils.structs import MinorVersion
+from perun.profile.factory import Profile
+
+
+def get_machine_info(machine_info: Optional[str] = None) -> dict[str, Any]:
+    """Returns machine info either from input file or constructs it from environment
+
+    :param machine info: file in json format, which contains machine specification
+    :return: parsed dictionary format of machine specification
+    """
+    if machine_info is not None:
+        with open(machine_info, "r") as machine_handle:
+            return json.load(machine_handle)
+    else:
+        return environment.get_machine_specification()
+
+
+def get_param(cfg: dict[str, Any], param: str, index: int) -> Any:
+    """
```

Comment:
The docstring is missing a short description. The function seems rather simple and understandable on its own, but still, should we add at least a short description here?